src/magic_folder/test/test_web.py

```diff
@@ -463,6 +472,86 @@ def test_list_folders(self, folders):
         )
 
 
+
+class RedirectTests(SyncTestCase):
+    """
+    Test for handling redirects from werkzeug.
+    """
+    url = DecodedURL.from_text(u"http://example.invalid./v1/magic-folder")
+
+    @skipIf(
+        StrictVersion(werkzeug.__version__) < StrictVersion("1.0.0"),
```

Comment:
The version in the nixos snapshot we are using is < 1.

Reply:
We should file a bug to upgrade it then .. and/or figure out some wider strategy about how to ensure Nix can track our Python requirements since I suspect this will be the case (i.e. "Nix has old version") for almost anything we want to upgrade..?

(A comment would be good here indicating that too...)

Reply:
#409

Reply:
Maybe after the current round of priorities is out of the way, we can port this codebase to Python 3 and then a lot of Python packaging/dependency issues probably become simpler?  They certainly do wrt Nix.